src/app/src/index.js

```diff
@@ -7,12 +7,35 @@ import { Provider } from 'react-redux';
 import './sass/main.scss';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
-import * as appActions from './app.actions';
+import {
+    hideIntro,
+    selectSensor,
+    deselectSensor,
+    showSensorModal,
+    hideSensorModal,
+    startPollingSensor,
+    failPollingSensor,
+    completePollingSensor,
```

Comment:
Setting the `sensorData` along with `completePollingSensor` instead of `setSensorData` has a small conflict of logic in #63 where we can set default values for the sensor, if polling sensors fail. In that case, we'd have to call `completePollingSensor` and `failPollingSensor`, which is confusing.

Reply:
I merged `setSensorData` into `completePollingSensor` while trying to figure out why I was running into issues adding new actions (trying to reduce the amount of code that could be causing problems). I looked to our other projects to see how we've set these async actions up in the past. We've used a `START/COMPLETE/FAIL` pattern ([examples](https://github.com/azavea/american-water-tank-management/blob/221caa657e56adab0ff44f2e664a428ef611ad49/src/app/js/actions.notes.js#L13-L27)), where the `complete` action sets the data. It may be that we need to move away from that in this project, but does it make sense to discuss things over on #63? I can revert things here if it will cause too many problems when resolving conflicts, though.

Reply:
Sure, let's move convo to #63. It may be that we initialize with default values on app load, which would allow us to keep the `START/COMPLETE/FAIL` pattern.  We can call the changes good here -- one more conflict on this pile doesn't make a dent.

Reply:
Cool. 

>  It may be that we initialize with default values on app load

That makes sense to me.